README.md

```diff
@@ -4,9 +4,9 @@
 
 ![Build Status](https://travis-ci.org/legatoproject/legato-af.svg)
 
-Legato Open Source Project started out as an initiative by Sierra Wireless Inc. to provide an open, secure and easy to use Application Framework to lower the barrier for developers, who are not familiar with embedded programming, to participate in the exponential growth of "Internet of Things". Visit [legato.io](legato.io) to learn more or visit the [Legato forum](forum.legato.io).
+Legato Open Source Project is an initiative by Sierra Wireless Inc. which provides an open, secure and easy to use Application Framework for embedded devices. The project enables developers who are not experienced in traditional embedded programming to participate in the exponential growth of "Internet of Things". Visit [legato.io](http://legato.io) to learn more or visit the [Legato forum](forum.legato.io).
```

Comment:
The forum link needs to be updated as well. It should be:
`... [Legato forum](https://forum.legato.io).`